### ISSUE TITLE
[Bug]: [Bug Bash] The hovered-on menu item background color should be grey in VS Dark theme

### ISSUE THREAD

**Issue by v-crchang** [NuGet/Home#12092]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

Dev\6.4.0.85

### Impact

It bothers me. A fix would be nice

## Repro steps:  
1. Launch VS and create a solution [e.g. C# Console App (.NET Framework 4.8.1)].
2. Go to Tools | Options… | Environment | General to change the Color Theme to Dark, click “OK”.
3. Open the PM UI, go to “Browse” tab, select a package in the package list, and click the arrow-down button of “Version” and “Package source” drop-down box.
4. Check the background color when hovering on the menu item.

## Expected:
The background color when hovering on the menu item should be grey in VS Dark theme.
## Actual: 
The search-box hovered-on menu has a grey background color, but “Version” and “Package source” hovered-on menu does not as below screenshot.
![image](https://user-images.githubusercontent.com/71433406/190374283-8c9833f0-78bf-44ab-a5da-3a0549502da2.png)

## Notes:
1. It only occurs in Dark theme.
2. The issue starts from Dev\6.4.0.58.

**Comment by donnie-msft** [comment]
Blocked on External bug: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1600486/
Noted in: https://github.com/NuGet/NuGet.Client/pull/4775

**Comment by v-luzh** [comment]
It still reproes in Blend theme on Blend for Visual Studio as below. Please fix them together after the blocking issue fixed.
![image](https://user-images.githubusercontent.com/20632625/191694640-bcfb4df3-d066-4113-b3ef-c948337c3d0b.png)

**Comment by donnie-msft** [comment]
External bug was resolved, so putting this in the backlog.

**Comment by donnie-msft** [comment]
The external bug fix has made its way through to PM UI, and I no longer repro this in 17.7.0 Preview 3.0 [33825.54.main]`.

Dark is now working properly:
![image](https://github.com/NuGet/Home/assets/49205731/4e25c18f-e65f-45cc-b4db-3c2c14b1b43c)
![image](https://github.com/NuGet/Home/assets/49205731/1ee76788-a87b-4f9d-a387-123cc663cbd5)

Don't see any regression in other themes, either
![image](https://github.com/NuGet/Home/assets/49205731/74fc4c0d-35b8-45c4-b85a-99f5cb057b1c)

@v-crchang could you please try this again and close if it's no repro for you as well? Thanks.

**Comment by v-luzh** [comment]
Hi, @donnie-msft, it is fixed on NuGet Client Dev\6.8.0.13 on VS. But there seems another issue on Blend for VS in Blend theme, the hovered item background colors are different as below. Could ypu please help to have a look if it should be fix in this bug or should we file another one against Blend theme? Thanks.
![image](https://github.com/NuGet/Home/assets/20632625/ce5b3a44-9f10-42a6-af7c-4c2333c2c8bd)

**Comment by donnie-msft** [comment]
@v-luzh awesome, thanks for verifying this is fixed! On that note, I will close this bug as it applies to the VS theming.

Thank you for also finding the Microsoft Blend bug. I've contacted the team responsible for those themes, and filed a bug for them to address it: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1849102
